tests/Integration/PrestaShopBundle/Command/FeatureFlagCommandTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace Integration\PrestaShopBundle\Command;
+
+use Doctrine\ORM\EntityManagerInterface;
+use PrestaShopBundle\Entity\FeatureFlag;
+use PrestaShopBundle\Entity\Repository\FeatureFlagRepository;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Tests\Resources\DatabaseDump;
+
+class FeatureFlagCommandTest extends KernelTestCase
+{
+    public static function setUpBeforeClass(): void
+    {
+        parent::setUpBeforeClass();
+        $kernel = self::bootKernel();
+        DatabaseDump::restoreTables(['feature_flag']);
+        $featureFlag = new FeatureFlag('test_feature_flag');
+        /** @var EntityManagerInterface $entityManager */
+        $entityManager = $kernel->getContainer()->get('doctrine.orm.entity_manager');
+        $entityManager->persist($featureFlag);
+        $entityManager->flush();
+    }
+
+    public static function tearDownAfterClass(): void
+    {
+        parent::tearDownAfterClass();
+        //DatabaseDump::restoreTables(['feature_flag']);
```

Comment:
```suggestion
```

Reply:
Good call! I commented it for a test I'll fix that ASAP